packages/fluentui/react-northstar/src/themes/teams/components/Chat/chatMessageStyles.ts

```diff
@@ -50,8 +50,7 @@ export const chatMessageStyles: ComponentSlotStylesPrepared<ChatMessageStylesPro
       wordWrap: 'break-word',
 
       ...((v.hasMention || v.isImportant) && {
-        '::before': {
-          content: '""',
+        [`& .${chatMessageSlotClassNames.bar}`]: {
           backgroundColor: v.hasMention ? v.hasMentionColor : v.isImportantColor,
           height: '100%',
           left: '0',
```

Comment:
Do any of these styles need to be applied still? Now that the pseudo element is gone, it may not be necessary.

Reply:
Yes, we still need all of them since we still want the new `div` to be positioned as the pseudo element was before.